frontend/plugins/operation_ui/src/modules/cycle/components/detail/CycleProgressChart.tsx

```
@@ -94,25 +101,73 @@ export const CycleProgressChart = ({
+
+export const ProgressChartGradients = ({
+  data = [],
+}: {
+  data: { date: string; started: number; completed: number; future: boolean }[];
+}) => {
+  const lines: ('started' | 'completed')[] = ['started', 'completed'];
+
+  return (
+    <defs>
+      {lines.map((line) => (
+        <linearGradient id={`${line}Gradient`} x1="0" y1="0" x2="1" y2="0">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)